src/core/grade_notifier.py

```diff
@@ -107,27 +107,28 @@ def create_text_message(change_log):
                 .newline()
             class_num += 1
 
-    new_message \
-        .newline() \
-        .add("Grade for those classes are:") \
-        .newline() \
-        .add("----------------------------") \
-        .newline()
-
-    for elm in change_log.classes:
-        if len(elm['grade']) != 0:
-            new_message \
-                .add("{0}: {1} (Grade) -- {2} (Grade Points)".format(
-                    elm['name'], elm['grade'], elm['gradepts'])) \
-                .newline()
-
-    if gpa.get_term_gpa() >= 0:
-
-        new_message.add("----------------------------") .newline() .add(
-            "Your term GPA is: {0}".format(
-                gpa.get_term_gpa())) .newline() .add(
-            "Your cumulative GPA is: {0}".format(
-                gpa.get_cumulative_gpa())) .newline()
+    if omitgrade == "False" or omitgrade != "True":
```

Comment:
Not a fan of omitgrade being a string acting as a boolean. Let's make it a boolean outright.